File: cmd/logout.go
package cmd

import (
	"fmt"
	"os"

	"github.com/spf13/cobra"

	"github.com/pcr-developers/cli/internal/auth"
)

var logoutCmd = &cobra.Command{
	Use:   "logout",
	Short: "Remove saved credentials",
	Run: func(cmd *cobra.Command, args []string) {
		auth.Clear()
		fmt.Fprintln(os.Stderr, "PCR: Logged out.")
	},
}

File: crates/pcr-core/src/commands/logout.rs
//! `pcr logout`. Mirrors `cli/cmd/logout.go`.

use crate::agent::OutputMode;
use crate::auth;
use crate::display;
use crate::exit::ExitCode;

pub fn run(_mode: OutputMode) -> ExitCode {
    auth::clear();
    display::eprintln("PCR: Logged out.");
    ExitCode::Success
}
